tests/ui/cfg/issue-112674.rs

```diff
@@ -0,0 +1,11 @@
+pub use a::x;
```

Comment:
can you rename the test to align it with the other tests for that diagnostic? either a new test with a similar name or just putting this into the existing tests